meta-hpe/meta-common/recipes-phosphor/vehci/vehci-gxp-b0-workarround/udc-reconnect.sh

```diff
@@ -0,0 +1,37 @@
+#~/bin/sh
```

Comment:
```suggestion
#!/bin/sh
```
Right?

Reply:
Fixed in 0c5e1d5762067f088f7b84c828f53ca9369a33c9